packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js

```diff
@@ -86,6 +87,13 @@ export function PatternCategoryPreviews( {
 					return true;
 				}
 
+				if (
+					category.name === starterPatternsCategory.name &&
+					pattern.blockTypes?.includes( 'core/post-content' )
+				) {
+					return true;
+				}
+
```

Comment:
It's not great that the categorization logic is duplicated in two components. Feels like something that can be factorized. Maybe within `use-patterns-state`